packages/blaze/package-types.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
You can remove the package-types.json file. It is only needed if the package depends on `zodern:types`. Since blaze doesn't, this file isn't included when the package is published.